okapi/services/fieldnotes/upload/docs.xml

```diff
@@ -0,0 +1,49 @@
+<xml>
+    <brief>Upload Fieldnotes</brief>
+    <issue-id>630</issue-id>
+    <desc>
+        <p>Upload a set of one or more fieldnotes records.</p>
+    </desc>
+    <req name='field_notes'>
+        <p>Fieldnotes consist of one or more records in CSV format. Each
+        record comprises a geocache log consisting of four fields:</p>
```

Comment:
CSV format specification mentions an optional header. Should we specify if this header is required, optional, or forbidden here? (It would be much easier if there was some external specification for the "fieldnotes file", but you mentioned there is no such specification, so we may need to specify all such details ourselves.)